save-frontend/src/main/kotlin/com/saveourtool/save/frontend/components/views/index/CardUser.kt

```diff
@@ -55,24 +53,27 @@ val cardUser: FC<IndexViewProps> = FC { props ->
     val (countVulnerability, setCountVulnerability) = useState(0)
     val navigate = useNavigate()
 
+    @Suppress("TOO_MANY_LINES_IN_LAMBDA")
     useRequest {
-        val organizationsNew: List<OrganizationDto> = get(
-            "$apiUrl/organizations/get/list-by-user-name?userName=${props.userInfo?.name}",
-            jsonHeaders,
-            loadingHandler = ::loadingHandler,
-        )
-            .decodeFromJsonString()
-
-        setOrganizations(organizationsNew)
-
-        val countVuln: Int = get(
-            "$apiUrl/vulnerabilities/count-by-user?userName=${props.userInfo?.name}",
-            jsonHeaders,
-            loadingHandler = ::loadingHandler,
-        )
-            .decodeFromJsonString()
-
-        setCountVulnerability(countVuln)
+        props.userInfo?.name?.let {
+            val organizationsNew: List<OrganizationDto> = get(
+                "$apiUrl/organizations/get/list-by-user-name?userName=$it",
+                jsonHeaders,
+                loadingHandler = ::loadingHandler,
+            )
+                .decodeFromJsonString()
```

Comment:
Probably `unsafeCast` is needed (maybe not)

Reply:
Did you mean ```unsafeMap``` ?
I don't think it's needed here.